pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.ejs

```diff
@@ -0,0 +1,82 @@
+      data-max-points="<%= assessment_question.max_points %>">
+  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+  <input type="hidden" name="modified_at" value="<%= instance_question.modified_at %>">
+  <div class="form-group row justify-content-center">
+    <label class="custom-control-inline col-auto mx-0">
+      <span class="">Points</span>
+      <div class="custom-control custom-switch mx-2">
+        <input class="custom-control-input js-manual-grading-pts-perc-select"
+               name="use_score_perc" type="checkbox">
+        <span class="custom-control-label">Percentage</span>
+      </div>
+    </label>
+  </div>
+  <%- include('gradingPointsSection',
+    { type: 'manual', type_label: 'Manual',
+      points: locals.manual_points ?? instance_question.manual_points,
+      max_points: assessment_question.max_manual_points,
+      rubric: locals.rubric_data_manual,
+      rubric_grading: submission.rubric_grading_manual
+  }); %>
+  <% if (assessment_question.max_auto_points > 0 || (locals.auto_points ?? instance_question.auto_points) > 0) { %>
+    <%- include('gradingPointsSection',
+      { type: 'auto', type_label: 'Auto',
+        points: locals.auto_points ?? instance_question.auto_points,
+        max_points: assessment_question.max_auto_points,
+        rubric: locals.rubric_data_auto,
+        rubric_grading: submission.rubric_grading_auto
+    }); %>
+    <div class="form-group js-manual-grading-points w-100">
+      Total Points:
+      <span class="float-right">
+        <span class="js-value-total-points"><%= Math.round(100 * (locals.points ?? instance_question.points)) / 100 %></span> /
+        <%= assessment_question.max_points %>
+      </span>
+    </div>
+    <div class="form-group js-manual-grading-percentage w-100">
+      Total Score:
+      <span class="float-right">
+        <span class="js-value-total-percentage"><%= Math.round(100 * (locals.score_perc ?? instance_question.score_perc)) / 100 %></span>%
+      </span>
+    </div>
+  <% } %>
+  <div class="form-group">
+    <label>Feedback:
+      <textarea name="submission_note" class="form-control js-submission-feedback"
+                style="min-height: 1em;"
+        <% if (locals.disable || !authz_data.has_course_instance_permission_edit) { %>readonly<% } %>
+                aria-describedby="submission-feedback-help-<%= context %>"><%= submission.feedback?.manual %></textarea>
+        <small id="submission-feedback-help-<%= context %>" class="form-text text-muted">
+          Markdown formatting, such as *emphasis* or `code`, is permitted
+          and will be used to format the feedback when presented to the
+          student.
+        </small>
+    </label>
+  </div>
+  <div class="text-right">
+    <% if (!locals.disable && authz_data.has_course_instance_permission_edit) { %>
+      <button type="submit" class="btn btn-primary" name="__action" value="add_manual_grade">Submit</button>
```

Comment:
When I click "Submit" I get the following error on the JS console:
```
An invalid form control with name='score_manual_percent' is not focusable.
```

Reply:
This should be resolved, please let me know if this issue persists after the latest changes.